whetstone/compiler-test/src/test/kotlin/com/freeletics/mad/whetstone/codegen/TestHelper.kt

```diff
@@ -48,26 +49,30 @@ private fun compile(fileName: String, source: String, output: String) {
     assertThat(compilation.compile().exitCode).isEqualTo(ExitCode.OK)
 }
 
+@OptIn(ExperimentalCompilerApi::class, ExperimentalAnvilApi::class)
 private fun compileWithAnvil(fileName: String, source: String, output: String) {
-    val compilation = KotlinCompilation().apply {
-        configure()
+    val compilation = AnvilCompilation().apply {
         configureAnvil()
+        kotlinCompilation.configure()
+        useIR(true)
```

Comment:
This is a helper from Anvil which was previously not working for us. It has its own `configureAnvil` method so that we don't need our own below anymore